### ISSUE TITLE
特定专业领域知识

### ISSUE THREAD

**Issue by cresting1222** [Laoyu84/4onebench#1]
面向特定专业领域这种多步复杂推理，目前看没有什么好的解决措施，我们目前还是通过KG来承载人工经验，让大模型在人的经验下游走、发散推理（https://github.com/codefuse-ai/CodeFuse-muAgent）

**Comment by Laoyu84** [comment]
Agree. It's pretty difficult for now to build a functional general-purpose agent, which can accomplish any arbitrary tasks.
